.github/workflows/main.yaml

```diff
@@ -0,0 +1,345 @@
+on: 
+  push: 
+    tags:
+     - '*'
+
+jobs:
+  get-artifact-list:
+    runs-on: ubuntu-latest
+    steps:
+      - name: get the list
+        run: |
+         git clone https://github.com/amshinde/kata-packaging
+         pushd kata-packaging
+         tag=`echo $GITHUB_REF | cut -d/ -f3-`
+         git checkout $tag
+         popd
+         ./kata-packaging/artifact-list.sh > artifact-list.txt
```

Comment:
Is `$PWD` guaranteed to be writable here?

Reply:
Yes, this environment is writable.